temporian/core/event_set_ops.py

```diff
@@ -1416,6 +1414,100 @@ def leak(self: EventSetOrNode, duration: Duration) -> EventSetOrNode:
 
         return leak(self, duration=duration)
 
+    def map(
+        self: EventSetOrNode,
+        func: MapFunction,
+        output_dtypes: Optional[TargetDtypes] = None,
+    ) -> EventSetOrNode:
+        """Applies a function on each of an [`EventSet`][temporian.EventSet]'s
+        values.
```

Comment:
Nit: How to make it clear that a "value" is a feature-value for a single event?

Reply:
reworded to "Applies a function on each value of an [`EventSet`][temporian.EventSet]'s features."

Reply:
Nice.

Could you also add that:
- This function will be slow. When possible existing operators should be used.
- A temporian graph with a map cannot be serialized.